developer/src/kmc/README.md

```diff
@@ -1,105 +1,191 @@
-Keyman Developer - Next Generation Compiler
+Keyman Developer - Command Line Compiler
 ================
 
-This package provides the following Keyman **command line tools**:
+This package provides a command-line interface to the Keyman Developer compiler
+toolchain, `kmc`.
 
- - `kmc` — takes **LDML Keyboard .xml sources** and compiles them in to a
-   KMXPlus **.kmx** file.
- - `kmlmc` — takes **lexical model sources** and compiles them in to a **.js**
-   file.
- - `kmlmp` — uses a `.model.kmp` file to generate a redistributable **lexical
-   model package**.
+## Install kmc
 
-`kmlmc` is intended to be used standalone, or as part of a build system. `kmlmp`
-is used only by command line tools.
+`kmc` is available as:
+* a part of Keyman Developer (Windows)
+* an npm package, and
+* a zip download
 
-Note: `kmc` will in the future replace `kmlmc` and `kmlmp`.
+Hint: Unlike previous versions of Keyman Developer, version 17 of kmc does not
+require WINE to run the command line tools on Linux or macOS.
 
-In order to build [lexical models][], these tools must be built and compiled.
+### Keyman Developer integration (Windows only)
 
-[lexical models]: https://github.com/keymanapp/lexical-models
+kmc is included with a default installation of Keyman Developer, including a
+runtime of node.js, and will be on the system path by default. No additional
+configuration or installation is required.
 
+### npm (Windows, macOS, and Linux)
 
-Install
--------
+kmc is also available as an npm package,
+[@keymanapp/kmc](https://npmjs.com/package/@keymanapp/kmc).
 
-Install `kmc` globally:
+You'll need [node.js](https://nodejs.org/), version 18.0 or later.
```

Comment:
Is there anything to install for WASM support?  mentioned in some readme's above for the uset parser.

Reply:
No additional components are required -- just a reasonably modern version of Node, which this is.